Sources/Action/UIKitExtensions/UIBarButtonItem+Action.swift

```diff
@@ -37,5 +38,28 @@ public extension Reactive where Base: UIBarButtonItem {
             }
         }
     }
+    public func bindToAction <Input,Output>(_ action:Action<Input,Output>?,_ inputTransform: @escaping (Base) -> (Input))   {
+        
+        self.base.resetActionDisposeBag()
+        
+        if (action == nil) {
```

Comment:
This could be a guard, like `guard let action = action`, then we don't need to force-unwrap underneath either.